Configurations/10-main.conf

```diff
@@ -965,10 +965,76 @@ sub vms_info {
         # with previous MIPS ISA versions, in sense that unlike
         # prior versions original MIPS binary code will fail.
         #
-        inherit_from     => [ "android64", asm("mips64_asm") ],
+        inherit_from     => [ asm("mips64_asm") ],
         perlasm_scheme   => "64",
     },
 
+    # GCC targets
+    "android" => {
+        inherit_from     => [ "android-base" ],
+        cflags           => add(picker(default => "-mandroid")),
+    },
+    "android-x86" => {
+        inherit_from     => [ "android", "android-x86-base" ],
+    },
+    "android-armeabi" => {
+        inherit_from     => [ "android", "android-armeabi-base" ],
+    },
+    "android-mips" => {
+        inherit_from     => [ "android", "android-mips-base" ],
+    },
+    "android64" => {
+        inherit_from     => [ "android64-base" ],
+        cflags           => add(picker(default => "-mandroid")),
+    },
+    "android64-aarch64" => {
+        inherit_from     => [ "android64", "android64-aarch64-base" ],
+    },
+    "android64-x86_64" => {
+        inherit_from     => [ "android64", "android64-x86_64-base" ],
+    },
+    "android64-mips64" => {
+        inherit_from     => [ "android64", "android64-mips64-base" ],
+    },
+
+    # Clang targets
+    "android-clang" => {
+        inherit_from     => [ "android-base" ],
+        cc               => "clang",
+        cxx              => "clang++",
+        cflags           => add(picker(default => "-gcc-toolchain \$(GCC_TOOLCHAIN) -fno-integrated-as")),
```

Comment:
As for `-fno-integrated-as`. Is it actually so that *all* platforms "require" it? "Require" in quotes is reference to the fact that there was bug reported with ARM (right?) integrated clang assembler. If only single platform is problematic, then I'd argue that only that one should get the flag...